src/main.cpp

```diff
@@ -3004,6 +3004,22 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, CBlockIn
         }
     }
 
+    // Coinbase transaction must include founders' reward if the
+    // block height is below 210000, which is the first halvening.
+    // The genesis block is excused from this requirement.
+    if ((nHeight > 0) && (nHeight < FOUNDERS_REWARD_UNTIL_BLOCK)) {
```

Comment:
That won't be the case anyway because of the genesis block, but I want to avoid being inclusive here since we want to turn the founders reward off *before* the rewards halve.